ci-operator/step-registry/ipi/conf/aws/sharednetwork/ipi-conf-aws-sharednetwork-commands.sh

```diff
@@ -32,7 +32,7 @@ fi
 STACK_NAME="${CLUSTER_NAME}-shared-vpc"
 aws --region "${REGION}" cloudformation create-stack \
   --stack-name "${STACK_NAME}" \
-  --template-body "$(cat /tmp/01_vpc.yaml)" \
+  --template-body "$(cat "/tmp/01_vpc.yaml")" \
```

Comment:
need the `"` here?

Reply:
I remember that adding `"` is because I met a template format issue https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29050/rehearse-29050-periodic-ci-openshift-verification-tests-master-installer-rehearse-installer-rehearse-aws-qe/1532215660975755264

Reply:
hmm, I guess that is some flake. Can we remove it for one more try.

Reply:
@jianlinliu  this must be a mistake, this file should not be changed, the ref is not used in workflows. Reverted.
Thanks